docs/apache-airflow-providers-amazon/operators/transfer/s3_to_sql.rst

```diff
@@ -0,0 +1,75 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+============================
+Amazon S3 to SQL
+============================
```

Comment:
```suggestion
==============
Amazon S3 to SQL
==============
```

Reply:
This results in an error when building the docs:

```
------------------------------ Error   6 --------------------
 WARNING: Title overline too short.

File path: apache-airflow-providers-amazon/operators/transfer/s3_to_sql.rst (18)

  13 |     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
  14 |     KIND, either express or implied.  See the License for the
  15 |     specific language governing permissions and limitations
  16 |     under the License.
  17 | 
> 18 | ==============
  19 | Amazon S3 to SQL
  20 | ==============
  21 | 
  22 | Use the ``S3ToSqlOperator`` transfer to copy data from an Amazon Simple Storage Service (S3) file into an existing
  23 | SQL table.
------------------------------ Error   7 --------------------
 WARNING: Title underline too short.

File path: apache-airflow-providers-amazon/operators/transfer/s3_to_sql.rst (37)

  32 | ---------
  33 | 
  34 | .. _howto/operator:S3ToSqlOperator:
  35 | 
  36 | Amazon S3 To SQL Transfer Operator
> 37 | ============================
  38 | 
  39 | To get more information about this operator visit:
  40 | :class:`~airflow.providers.amazon.aws.transfers.s3_to_sql.S3ToSqlOperator`
  41 | 
  42 | Example usage:

############################## End docs build errors summary ##############################



```